README.md

```diff
@@ -36,14 +59,38 @@ Return to your original locations.  In this step all groups will act like Design
 
 ***Q1.3 Briefly describe your solutions (use a list) and any strengths/weaknesses you see.***
 
+Solution for Safekeeping
+
+|         |         |           |
+| ------------- |:-------------:| -----:|
+| Function    | We can put a tracker in the wallet or somewhere else which need to Safekeeping, when the tracker is always from your wallet that suddenly will alarm|  |
```

Comment:
Good! If a tracker could somehow be embedded in the leather (or whatever material) of the wallet, syncing the tracker with a mobile application on your phone could help users find their lost items.